images-list

```diff
@@ -1923,7 +1923,7 @@ registry.k8s.io/kube-state-metrics/kube-state-metrics rancher/mirrored-kube-stat
 registry.k8s.io/kube-state-metrics/kube-state-metrics rancher/mirrored-kube-state-metrics-kube-state-metrics v2.6.0
 registry.k8s.io/kube-state-metrics/kube-state-metrics rancher/mirrored-kube-state-metrics-kube-state-metrics v2.10.1
 registry.k8s.io/kube-state-metrics/kube-state-metrics rancher/mirrored-kube-state-metrics-kube-state-metrics v2.12.0
-registry.k8s.io/kube-state-metrics/kube-state-metrics rancher/mirrored-kube-state-metrics-kube-state-metrics v2.13.0
+registry.k8s.io/kube-state-metrics/kube-state-metrics rancher/mirrored-kube-state-metrics-kube-state-metrics v2.14.0
```

Comment:
Do not change/remove existing tags. Only add.

Reply:
Oh geeze - great catch Brad! My copy-paste job must have been sloppy. Just fixed this.